edge-modules/ModuleRestarter/ModuleRestarter.csproj

```diff
@@ -0,0 +1,65 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup Condition="'$(DotNet_Runtime)' != 'netcoreapp3.0'">
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(OS)|$(DotNet_Runtime)' == 'Unix|netcoreapp3.0'">
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <Configurations>Debug;Release;CodeCoverage;CheckInBuild</Configurations>
+    <HighEntropyVA>true</HighEntropyVA>

```

Comment:
I remember Varun used this for a project (forget which project); but I forget why.  Do we really need this?

Reply:
I am not sure. I will try removing it, but it exists in all other csproj files under edge-modules as well.

Reply:
[The documentation](https://docs.microsoft.com/en-us/cpp/build/reference/highentropyva?view=vs-2019) says that this indicates support for ASLR with 64-bit addresses, which I'm guessing we support or it would not be elsewhere. Guessing we should keep it?

Reply:
[This documentation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/highentropyva-compiler-option) says:

The highentropyva compiler option tells the Windows kernel whether a particular executable supports high entropy Address Space Layout Randomization (ASLR).

If it is everywhere else I'm guessing we should keep it.

Reply:
Please check with Anca if she knows what this parameter is used for.  And btw you are right we used this parameter in many projects except loadGen and MessageAnalyzer, please make it consistent if needed.  thanks.

Reply:
This cpp Microsoft Docs [link](https://github.com/MicrosoftDocs/cpp-docs/issues/282#issuecomment-396361257) gives more insight into the option (purpose is same as for c#). I think we should keep it for security reasons so I will add the option where needed.